samples/dnn/classification.cpp

```diff
@@ -141,28 +148,52 @@ int main(int argc, char** argv)
         net.setInput(blob);
         //! [Set input blob]
         //! [Make forward pass]
-        Mat prob = net.forward();
-        //! [Make forward pass]
-
-        //! [Get a class with a highest score]
-        Point classIdPoint;
+        double t_sum = 0.0;
+        double t;
```

Comment:
> `double t;`

Prefer to declare variables near its usage. It significantly increases code readability.

Reply:
Fixed.